backend/judging-algorithm/parser-functions.ts

```diff
@@ -0,0 +1,61 @@
+import axios from "axios";
+import https from 'https';
+import { parseJudgeCSV, parseHackerTeamCSV, parseRoomsCSV, parseRotationTimeCSV } from  "./parser";
+import 'dotenv/config';
+
+export async function updateJudgeData() {
+    try {
+        await axios.delete(`${process.env.SERVER_URL}/judges`);
+
+        const newJudges = parseJudgeCSV('./data/csv_inputs/judges.csv');
+
+        for (const judge of newJudges) {
+            await axios.post(`${process.env.SERVER_URL}/judges`, judge, {httpsAgent: new https.Agent({ keepAlive: true })})
```

Comment:
Can we create helper functions to handle deleting and posting data, dealing with the HTTP requests, and providing more informative console logs for success and error cases? This is so we can provide more context-specific error messages and handle errors in a more structured manner.